src/component/MealPlanner/Form.js

```diff
@@ -22,12 +23,12 @@ class MealForm extends Component {
               <Ingredient handleIngredients={this.props.handleIngredients} />
               <Steps handleSteps={this.props.handleSteps} />
               <Description handleDescription={this.props.handleDescription} />
-              <Diets handleDiet={this.props.handleDiet} />
-              <Allergy handleAllergy={this.props.handleAllergy} />
-              <Course handleCourse={this.props.handleCourse} />
-              <Cuisine handleCuisine={this.props.handleCuisine} />
+              <Attribute type='Diet' handleAttribute={this.props.handleDiet} />
```

Comment:
I also think that we can use one handler for this fields. we'll just need to put not only the data for `state`, but a string with attribute name.

Reply:
let's keep it as it is maybe. just create a new task and move our conversation there

Reply:
did you create a task for this feature, right?